dillo/web/posts/routes.py

```diff
@@ -326,17 +341,31 @@ def render_page():
             'max_results': '15'
             }, api=api)
 
-        # Populate the feed
-        for post in latest_posts._items:
-            author = post.user.full_name
-            updated = post._updated if post._updated else post._created
-            url = url_for('posts.view', post_shortcode=post.properties.shortcode, community_url=post.project)
-            content = post.properties.content[:500]
-            feed.add(post.name, str(content),
-                     content_type='html',
-                     author=author,
-                     url=url,
-                     updated=updated,
-                     published=post._created)
+        populate_feed(feed, latest_posts)
         return feed.get_response()
     return render_page()
+
+
+@blueprint.route('/p/feeds/<community_url>/latest.atom')
+def feeds_blog(community_url):
+    """Project feed generator for latest blogposts across a single project"""
+    # @current_app.cache.cached(60*5)
+    def render_page():
+        feed = AtomFeed('Dillo - Latest updates',
```

Comment:
Is this title printed anywhere in the feed? If so, could we use the default/main community name instead of 'Dillo'?

Reply:
Of course, done.